### PR TITLE
Render non-responsive hearing worksheet

### PR DESCRIPTION
**Pull request by slj**
Connects #2218

### CODE REVIEW

File: app/controllers/hearings/worksheets_controller.rb [hunk 1]
```diff
@@ -1,21 +0,0 @@
-class Hearings::WorksheetsController < HearingsController
```
Comment (sharonwarner) [inline app/controllers/hearings/worksheets_controller.rb]: Why did you decide to change this to WorksheetController? I think the Rails convention is to have plural controllers but singular models.

File: app/controllers/hearings/worksheet_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+class Hearings::WorksheetController < HearingsController
+  # TODO: remove this line once application_alt and application merge
+  layout "application_alt"
+
+  def show
+    @hearing_page_title = "Daily Docket | Hearing Worksheet"
+
+    respond_to do |format|
+      format.html { render template: "hearings/dockets/index" }
```
Comment (sharonwarner) [inline app/controllers/hearings/worksheet_controller.rb]: I don't think this hierarchy makes sense anymore if we're using the same twig file for worksheets and dockets. Let's move it to hearings/index.

File: app/controllers/hearings/worksheet_controller.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      format.json do
+        render json: hearing_worksheet(params[:id])
+      end
+    end
+  end
+
+  # Until the frontend makes a PUT request, code coverage is at risk, so...
```
Comment (sharonwarner) [inline app/controllers/hearings/worksheet_controller.rb]: Was this code coverage a problem before?

File: client/app/hearings/DailyDocketContainer.jsx [hunk 4]
```diff
@@ -21,6 +21,14 @@ export class DailyDocketContainer extends React.Component {
...
       this.props.getDockets();
     }
+
+    // Since the page title does not change when react router
+    // renders this component...
+    const pageTitle = document.getElementById('page-title');
+
+    if (pageTitle) {
+      pageTitle.innerHTML = ' | Daily Docket';
+    }
```
Comment (sharonwarner) [inline client/app/hearings/DailyDocketContainer.jsx]: This doesn't work if the user goes from the Daily Docket page back to Upcoming Hearing Days.

File: app/controllers/hearings/worksheet_controller.rb (hunk 3 above)
Comment (slj) [inline app/controllers/hearings/worksheet_controller.rb]: Beforehand, none of our code/feature tests were engaging the controller, so it was never being analyzed for coverage.

File: app/controllers/hearings/worksheets_controller.rb (hunk 1 above)
Comment (slj) [inline app/controllers/hearings/worksheets_controller.rb]: While working `config/routes.rb` I reasoned that it made sense to change it.  Didn't realize I broke convention.

File: app/controllers/hearings/worksheet_controller.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
+class Hearings::WorksheetController < HearingsController
```
Comment (sharonwarner) [inline app/controllers/hearings/worksheet_controller.rb]: This file should be deleted.

**Pull request merged**